[2021-10-02 04:10:37,843] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-02T03:57:02.410915+00:00 [queued]>
[2021-10-02 04:10:38,036] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.Training_Model_B 2021-10-02T03:57:02.410915+00:00 [queued]>
[2021-10-02 04:10:38,041] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:38,045] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-10-02 04:10:38,051] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-10-02 04:10:38,141] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): Training_Model_B> on 2021-10-02T03:57:02.410915+00:00
[2021-10-02 04:10:38,147] {standard_task_runner.py:52} INFO - Started process 17338 to run task
[2021-10-02 04:10:38,201] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'Training_Model_B', '2021-10-02T03:57:02.410915+00:00', '--job-id', '11', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp46uyafo0', '--error-file', '/tmp/tmp8zged6lm']
[2021-10-02 04:10:38,212] {standard_task_runner.py:77} INFO - Job 11: Subtask Training_Model_B
[2021-10-02 04:10:38,616] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.Training_Model_B 2021-10-02T03:57:02.410915+00:00 [running]> on host d6b4131456ad
[2021-10-02 04:10:38,899] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=Training_Model_B
AIRFLOW_CTX_EXECUTION_DATE=2021-10-02T03:57:02.410915+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-10-02T03:57:02.410915+00:00
[2021-10-02 04:10:38,902] {python.py:151} INFO - Done. Returned value was: 3
[2021-10-02 04:10:39,036] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=Training_Model_B, execution_date=20211002T035702, start_date=20211002T041037, end_date=20211002T041039
[2021-10-02 04:10:39,170] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-10-02 04:10:39,350] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
